.circleci/config.yml

```diff
@@ -439,7 +439,9 @@ workflows:
             branches:
               only:
                 - main
-      - algolia-index:
+  website:
+      jobs:
+        - algolia-index:
```

Comment:
```suggestion
    jobs:
      - algolia-index:
```

Reply:
The spacing didn't seem to stick from your local to this PR?

Reply:
I don't know how i can keep doing this wrong i am baffled